spec/rails_cursor_pagination/cursor_spec.rb

```diff
@@ -51,6 +51,45 @@
         expect(decoded.order_field_value).to eq record.author
       end
     end
+
+    context 'when ordering by created_at' do
```

Comment:
I would try to make this more generig
```suggestion
    context 'when ordering by a timestamp attribute' do
```